src/App.tsx

```diff
@@ -1,10 +1,49 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+const USER_ID = 12037;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState('all');
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  useEffect(
+    () => {
+      setErrorMessage(null);
+
+      getTodos(USER_ID)
+        .then((dataFromServer) => {
+          setTodos(dataFromServer);
+        })
```

Comment:
```suggestion
        .then(setTodos)
```
You can simplify in this case